### ISSUE TITLE
Project.save doesn't work

### ISSUE THREAD

**Issue by b4ldr** [gpocentek/python-gitlab#104]
Hi,

Im getting a strange error trying to save projects depending on if i get the project via Gitlab.projects.all() or Gitlab.projects.get().  if i use the former then i get the following failure 

```python
AttributeError                            Traceback (most recent call last)
<ipython-input-15-395e325a0c83> in <module>()
----> 1 p.save()

/usr/lib/python2.7/site-packages/gitlab/objects.pyc in save(self, **kwargs)
    330     def save(self, **kwargs):
    331         if self._from_api:
--> 332             self._update(**kwargs)
    333         else:
    334             self._create(**kwargs)

/usr/lib/python2.7/site-packages/gitlab/objects.pyc in _update(self, **kwargs)
    325             raise NotImplementedError
    326
--> 327         json = self.gitlab.update(self, **kwargs)
    328         self._set_from_dict(json)
    329
```

When inspecting the issue i see the following i see that the project returned from `Gitlab.projects.all()` as the gitlab attribute `gitlab.objects.ProjectManage` which has no update method however using the `Gitlab.projects.get()` method i the gitlab attribute is `gitlab.Gitlab` which does have an update method 

```python
In [20]: p1 = g.projects.get(1)

In [21]: p2 = g.projects.all()[0]

In [22]: p1
Out[22]: <gitlab.objects.Project at 0x179ac50>

In [23]: p2
Out[23]: <gitlab.objects.Project at 0x17a4c50>

In [24]: p1.gitlab
Out[24]: <gitlab.Gitlab at 0x16aaf50>

In [25]: p2.gitlab
Out[25]: <gitlab.objects.ProjectManager at 0x16bd510>

In [26]: p1.id
Out[26]: 4

In [27]: p2.id
Out[27]: 4
```

**Comment by gpocentek** [comment]
I think this has been fixed in commit 293a9dc. Could you check if this works better with the master branch?

**Comment by b4ldr** [comment]
Yes it looks fixed in master, thanks
